py3status/modules/fedora_updates.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-The number of package updates pending for a Fedora Linux installation.
+Display number of pending updates for Fedora Linux.
```

Comment:
Come up with something for...

A simple "Do Not Disturb" module that can turn on and off all system notifications. ??
Take a screenshot and optionally upload it to your online server. ??

Brb soon! Making coffee.